### PR TITLE
[BZ#2100766] Update BYOH docs for RHEL 8 openshift-ansible

### PR DESCRIPTION
**Pull request by yselkowitz**
Version(s):
4.11+
(A follow-up for 4.10 will follow separately.)

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2100766

Link to docs preview:
http://shell.lab.bos.redhat.com/~yselkowi/openshift-docs/bz2100766/machine_management/adding-rhel-compute.html#rhel-preparing-playbook-machine_adding-rhel-compute
http://shell.lab.bos.redhat.com/~yselkowi/openshift-docs/bz2100766/machine_management/more-rhel-compute.html#rhel-preparing-node_more-rhel-compute
http://shell.lab.bos.redhat.com/~yselkowi/openshift-docs/bz2100766/updating/updating-cluster-rhel-compute.html#rhel-compute-updating-minor_updating-cluster-rhel-compute

Additional information:
How to handle upgrades from 4.10 (with RHEL 7 ansible host) to 4.11 (with RHEL 8 ansible host) may need further consideration.

/cc @barbacbd @gpei

### CODE REVIEW

**Review by jldohmann — APPROVED** [review]
LGTM, here is the doc preview: http://file.rdu.redhat.com/jdohmann/yaakov-pr/machine_management/adding-rhel-compute#rhel-preparing-playbook-machine_adding-rhel-compute

**Comment by yselkowitz** [comment]
Another page needs to be modified for this
/hold

**Comment by yselkowitz** [comment]
QE has acked
/hold cancel

**Review by barbacbd — COMMENTED** [review]
/lgtm

**Review by jldohmann — COMMENTED** [review]
Mostly LGTM, I left a clarifying question below

File: modules/rhel-compute-updating.adoc [hunk 1]
```diff
@@ -44,32 +44,36 @@ Because the {op-system-base} machines require assets that are generated by the c
...
 .. On the machine that you run the Ansible playbooks, update the required packages, including `openshift-ansible`:
 +
 [source,terminal]
 ----
 # yum update openshift-ansible openshift-clients
 ----
++
+[IMPORTANT]
+====
+As of {product-title} 4.11, the Ansible playbooks are provided only for {op-system-base} 8.  If a {op-system-base} 7 system was used as a host for the {product-title} 4.10 Ansible playbooks, either the system must be upgraded, or a new system set up and inventories copied over.
```
[the remark this answers is not in the document]
Comment (yselkowitz) [reply modules/rhel-compute-updating.adoc]: Attempted to clarify; previews have been regenerated.

**Review by jldohmann — APPROVED** [review]

**Pull request merged**